src/main/java/seedu/bookmark/ui/BookCard.java

```diff
@@ -98,6 +118,18 @@ protected void initialize(Book book, int displayedIndex) {
         }
     }
 
+    private String determineGoalStyle(Book book) {
```

Comment:
I think this same method is in `DetailedBookCard` as well right? Can remove it from `DetailedBookCard` since it has no use there already

Reply:
Yup thanks for pointing it out! Forgot how DetailedBookCard inherits from BookCard